templates/search-results/search-results.js

```diff
@@ -43,4 +43,7 @@ export default async function decorate(doc) {
 
   if (searchBar) main.prepend(searchBar);
   main.append(section);
+
+  // Delete obsolete containers left behind
+  ['.filters-container', '.results-list-container', '.pagination-container'].forEach((selector) => main.querySelector(selector)?.remove());
```

Comment:
These three are added by content. Have you tried removing these blocks from the content instead, by chance? We may also need to adjust the template. If not, I'm ok with this little fix.